Day-02/java/thezexquex/Solution.java

```diff
@@ -0,0 +1,44 @@
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.*;
+
+public class Solution {
+
+    public static void main(String[] args) throws IOException {
+        var lines = Files.readAllLines(Path.of("index.txt")).stream()
```

Comment:
index.txt?

Reply:
I mean it works, but looks a bit odd and is inconsistent with the previous names

```suggestion
        var lines = Files.readAllLines(Path.of("input.txt")).stream()
```

Reply:
Uhm yeah, that's definitely not intentional